docs/ancestry.md

```diff
@@ -2071,7 +2071,7 @@ print(tree.tmrca(0,1))
 
 ```
 
-Because branch lengths are proportional to {math}`N^2` generations, other events (such as mutations or recombinations) will also be proportional to {math}`N^2`. For example, if c is set to 0, the dirac coalescent will behave as if under a Moran model. However if believed census size is inputted (e.g. {math}`N=10000`), a number of segregating sites proportional to {math}`N^2` will be obtained. Leading to unrealistic simulated data.
+Because branch lengths are proportional to {math}N^2 generations, the number of events simulated along branches (such as mutations or recombinations) will also be proportional to {math}N^2. For example, if a tree sequence generated using a given census size {math}N is used as input for simulating segregating sites (see {ref}Mutation simulations<sec_mutations> for details on how to do this), the expected number of segregating sites obtained will be proportional to {math}N^2.
```

Comment:
You're missing backticks here for the math expressions, ``{math}`N^2```

Reply:
My mistake. I will fix this.